packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -473,6 +485,9 @@ export default function PostOptionsMenu({
   post?.tags?.forEach((tag) => {
     if (tag.length) {
       const isBlocked = feedSettings?.blockedTags?.includes(tag);
+      if (isBlocked && isCustomFeed) {
```

Comment:
Added this because we don't remove all posts with the tag when it's blocked without refreshing, which then leaves the "follow" button if you click another post with this tag. Don't think this is desired behavior